presto-orc/src/test/java/com/facebook/presto/orc/TestCachingOrcDataSource.java

```diff
@@ -247,7 +248,8 @@ public void doIntegration(TestingOrcDataSource orcDataSource, DataSize maxMergeD
                 () -> { });
     }
 
-    private static class FakeOrcDataSource
+    @VisibleForTesting
+    static class FakeOrcDataSource
```

Comment:
Rename this to `DummyOrcDataSource`. Add a private constructor. Make this class standalone in test/`com.facebook.presto.orc`. Do this in a separate commit. But don't create a new pull request.